tmpfiles.d/tmp.conf

```diff
@@ -8,8 +8,8 @@
 # See tmpfiles.d(5) for details
 
 # Clear tmp directories separately, to make them easier to override
-D /tmp 1777 root root -
-#d /var/tmp 1777 root root 30d
+d /tmp 1777 root root 10d
+d /var/tmp 1777 root root 30d
```

Comment:
I wonder if we shouldn't be even more aggressive here, but this looks good for now